schema-transformation-rules/devworkspace/addRequiredFields.json

```diff
@@ -1,13 +1,13 @@
 [
   { 
     "op": "add",
-    "path": "/properties/spec/properties/commands/items/properties/exec/required/-",
```

Comment:
Why is this change required ?

Reply:
This patch was failing because `required` field is no longer present in `exec`